externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkSQLOperationManager.scala

```diff
@@ -52,13 +55,21 @@ class SparkSQLOperationManager private (name: String) extends OperationManager(n
 
   def getOpenSparkSessionCount: Int = sessionToSpark.size()
 
+  private lazy val operationModeDefault = getConf.get(OPERATION_PLAN_ONLY)
+
   override def newExecuteStatementOperation(
       session: Session,
       statement: String,
       runAsync: Boolean,
       queryTimeout: Long): Operation = {
     val spark = getSparkSession(session.handle)
-    val operation = new ExecuteStatement(spark, session, statement, runAsync, queryTimeout)
+
+    val operationModeStr =
+      spark.conf.get(OPERATION_PLAN_ONLY.key, operationModeDefault).toUpperCase(Locale.ROOT)
+    val operation = OperationModes.withName(operationModeStr) match {
+      case NONE => new ExecuteStatement(spark, session, statement, runAsync, queryTimeout)
```

Comment:
nit: do we need to consider the following case?
```
ser kyuubi.operation.plan.only.mode=abc;
select 1;
```

Reply:
would you like to make a follow up？

Reply:
Sorry, don't want to do.